app/dao/inbound_numbers_dao.py

```diff
@@ -4,7 +4,7 @@
 
 
 def dao_get_inbound_numbers():
-    return InboundNumber.query.all()
+    return InboundNumber.query.order_by(InboundNumber.updated_at, InboundNumber.number).all()
```

Comment:
I appreciate whenever my opinion is requested, but I am unfamiliar with the code and this have no preference in this case. Thank you.